arkane/common.py

```diff
@@ -64,27 +64,28 @@
 ################################################################################
 
 # Class dictionary for recreating objects from YAML files. This is needed elsewhere, so store as a module level variable
-ARKANE_CLASS_DICT = {'ScalarQuantity': ScalarQuantity,
-                     'ArrayQuantity': ArrayQuantity,
-                     'Conformer': Conformer,
-                     'LinearRotor': LinearRotor,
-                     'NonlinearRotor': NonlinearRotor,
-                     'KRotor': KRotor,
-                     'SphericalTopRotor': SphericalTopRotor,
-                     'HinderedRotor': HinderedRotor,
-                     'FreeRotor': FreeRotor,
-                     'IdealGasTranslation': IdealGasTranslation,
-                     'HarmonicOscillator': HarmonicOscillator,
-                     'TransportData': TransportData,
-                     'SingleExponentialDown': SingleExponentialDown,
-                     'Wilhoit': Wilhoit,
-                     'NASA': NASA,
-                     'NASAPolynomial': NASAPolynomial,
-                     'ThermoData': ThermoData,
-                     'np_array': np.array,
-                     'LevelOfTheory': LevelOfTheory,
-                     'CompositeLevelOfTheory': CompositeLevelOfTheory
-                     }
+ARKANE_CLASS_DICT = {
+    "ScalarQuantity": ScalarQuantity,
```

Comment:
I think for _this_ file we could just revert all the changes - I didn't change any source here. I could also presumably configure VSCode's Black to avoid this conversion. But more broadly I do think this is a good change for readability and maintenance considerations. Worth I think the extra scroll through the blame. Could be part of a larger discussion among the devs